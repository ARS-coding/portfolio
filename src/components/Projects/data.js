const FOODCHECK = {
  title: 'FoodCheck',
  type: 'Solo Training Project',
  status: 'MVP',
  githubUrl: 'https://github.com/tmansuroglu/Foodcheck',
  demoUrl1: 'https://foodcheck.netlify.app/',
  demoUrl2: '',
  iosUrl: '',
  androidUrl: '',
  goal: 'creating a quick and easy to use diet tracker',
  impactArr: [
    'Took responsibility of whole project when teammate was facing crisis and completed project before its deadline',
  ],
  skillsArr: [
    'HTML',
    'CSS',
    'JavaScript',
    'React',
    'Redux',
    'Firebase',
    'Ant Design',
    'Responsive Design',
    'Git',
    'Github',
    'ChartJS',
    'Routing',
  ],
  responsibilityArr: ['Practicing React'],
};

const JUNIPER = {
  title: 'Juniper',
  type: 'Training Project - Team Project',
  status: 'MVP',
  githubUrl: 'https://github.com/ReCoded-Org/capstone-istanbul-juniper',
  demoUrl1: 'https://juniper-ist.netlify.app/',
  demoUrl2: '',
  iosUrl: '',
  androidUrl: '',
  goal: 'teaching middle-eastern youth about environmental problems through games.',
  impactArr: [
    'Built signature game(memory game) of the project',
    "Elevated my teammates' work by organizing off-hour online meetings",
    'Became problem solver of the team',
  ],
  skillsArr: [
    'HTML',
    'CSS',
    'JavaScript',
    'React',
    'Jest',
    'Firebase',
    'AGILE Methodologies',
    'Ant Design',
    'Responsive Design',
    'Git',
    'Github',
    'Figma',
    'Routing',
    'i18n',
  ],
  responsibilityArr: [
    'Developing a memory game',
    'Code reviewing',
    'Bug bashing',
    'Developing games page',
    'Scrum master for 1 week',
    'Developing twitter section of homepage',
  ],
};

const BIZIM_TARIFLER_WEB = {
  title: 'Bizim Tarifler Web',
  type: 'Professional Project - Team Project',
  status: 'Delivered Project',
  githubUrl: '',
  demoUrl1: 'https://bizimtarifler.com/',
  demoUrl2: '',
  iosUrl: '',
  androidUrl: '',
  goal: 'Combining cooking and ingredient shopping by allowing users to browse recipes through the site and buy ingredients from the partner.',
  impactArr: ['Hmmm'],
  skillsArr: [
    'HTML',
    'CSS',
    'JavaScript',
    'VueJS',
    'NuxtJS',
    'Sass',
    'Git',
    'Github',
    'AGILE Methodologies',
    'Responsive Design',
  ],
  responsibilityArr: ['Bug bashing'],
};

const BIZIM_TARIFLER_MOBILE = {
  title: 'Bizim Tarifler Mobile',
  type: 'Professional Project - Team Project',
  status: 'Delivered Project',
  githubUrl: '',
  demoUrl1: '',
  demoUrl2: '',
  iosUrl: 'https://apps.apple.com/tr/app/bizim-tarifler/id1358778615?l=en',
  androidUrl:
    'https://play.google.com/store/apps/details?id=com.bizimtarifler.app',
  goal: 'Combining cooking and ingredient shopping by allowing users to browse recipes through the site and buy ingredients from the partner.',
  impactArr: ['Hmmm'],
  skillsArr: [
    'React Native',
    'React',
    'JavaScript',
    'Git',
    'Github',
    'AGILE Methodologies',
    'Responsive Design',
    'Bare React Native',
    'Managed Expo',
    'Firebase',
  ],
  responsibilityArr: ['Bug bashing'],
};

const WIDIO_WEB = {
  title: 'Widio Web',
  type: 'Professional Project - Team Project',
  status: 'Work In Progress',
  githubUrl: '',
  demoUrl1: 'https://widio.com/',
  demoUrl2: '',
  iosUrl: '',
  androidUrl: '',
  goal: 'Creating a property and vehicle sale site in which all ads are based on videos',
  impactArr: ['Hmmm'],
  skillsArr: [
    'HTML',
    'CSS',
    'JavaScript',
    'React',
    'NextJS',
    'Redux',
    'Ant Design',
    'Git',
    'Github',
    'AGILE Methodologies',
    'Responsive Design',
  ],
  responsibilityArr: ['Bug bashing'],
};

const ECEM_HAKAN_WEB = {
  title: 'Nutritonist Web Site',
  status: 'MVP - Work In Progress',
  type: 'Professional Project - Solo Project',
  githubUrl: '',
  demoUrl1: 'https://www.ecemhakan.com/',
  demoUrl2: '',
  iosUrl: '',
  androidUrl: '',
  goal: 'Creating a web presence for a nutritionist',
  impactArr: ['Hmmm'],
  skillsArr: [
    'HTML',
    'CSS',
    'JavaScript',
    'Vue',
    'NuxtJS',
    'Sass',
    'Bootstrap',
    'Git',
    'Github',
    'Responsive Design',
  ],
  responsibilityArr: ['Bug bashing'],
};

const PROJECTS_ARR = [
  BIZIM_TARIFLER_WEB,
  BIZIM_TARIFLER_MOBILE,
  WIDIO_WEB,
  ECEM_HAKAN_WEB,
  JUNIPER,
  FOODCHECK,
];

export default PROJECTS_ARR;
