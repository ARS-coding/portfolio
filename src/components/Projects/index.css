.projectIcon {
    margin: 40% 30% 30% 30%;
    transform: scale(1.5);
}

.projectContent {
    color: black;
}
