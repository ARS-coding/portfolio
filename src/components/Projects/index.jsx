import React from 'react';
import './index.css';
import {
    VerticalTimeline,
    VerticalTimelineElement,
} from 'react-vertical-timeline-component';

const Projects = () => {
    return (
        <>
            <h1 className='homeTitle' id='projects'>
                Projects
            </h1>
            <VerticalTimeline>
                <VerticalTimelineElement
                    className='vertical-timeline-element--work'
                    contentStyle={{
                        background: 'white',
                        color: 'black',
                    }}
                    contentArrowStyle={{
                        borderRight: '7px solid  white',
                    }}
                    date='Oct-Nov 2020'
                    iconStyle={{
                        background: 'white',
                    }}
                    icon={<i class='fas fa-project-diagram projectIcon'></i>}
                >
                    <h3 className='vertical-timeline-element-title'>Juniper</h3>
                    <h4 className='vertical-timeline-element-subtitle'>
                        Miami, FL
                    </h4>
                    <p>
                        Creative Direction, User Experience, Visual Design,
                        Project Management, Team Leading
                    </p>
                </VerticalTimelineElement>
                <VerticalTimelineElement
                    className='vertical-timeline-element--work'
                    contentStyle={{
                        background: 'white',
                        color: 'black',
                    }}
                    contentArrowStyle={{
                        borderRight: '7px solid  white',
                    }}
                    date='Oct-Nov 2020'
                    iconStyle={{
                        background: 'white',
                    }}
                    icon={<i class='fas fa-project-diagram projectIcon'></i>}
                >
                    <h3 className='vertical-timeline-element-title'>
                        FoodCheck
                    </h3>
                    <h4 className='vertical-timeline-element-subtitle'>
                        Miami, FL
                    </h4>
                    <p>
                        Creative Direction, User Experience, Visual Design,
                        Project Management, Team Leading
                    </p>
                </VerticalTimelineElement>
                <VerticalTimelineElement
                    className='vertical-timeline-element--work'
                    contentStyle={{
                        background: 'white',
                        color: 'black',
                    }}
                    contentArrowStyle={{
                        borderRight: '7px solid  white',
                    }}
                    date='Oct-Nov 2020'
                    iconStyle={{
                        background: 'white',
                    }}
                    icon={<i class='fas fa-project-diagram projectIcon'></i>}
                >
                    <h3 className='vertical-timeline-element-title'>
                        TMDB Project
                    </h3>
                    <h4 className='vertical-timeline-element-subtitle'>
                        Miami, FL
                    </h4>
                    <p>
                        Creative Direction, User Experience, Visual Design,
                        Project Management, Team Leading
                    </p>
                </VerticalTimelineElement>
                <VerticalTimelineElement
                    className='vertical-timeline-element--work'
                    contentStyle={{
                        background: 'white',
                        color: 'black',
                    }}
                    contentArrowStyle={{
                        borderRight: '7px solid  white',
                    }}
                    date='Oct-Nov 2020'
                    iconStyle={{
                        background: 'white',
                    }}
                    icon={<i class='fas fa-project-diagram projectIcon'></i>}
                >
                    <h3 className='vertical-timeline-element-title'>
                        E-Commerce
                    </h3>
                    <h4 className='vertical-timeline-element-subtitle'>
                        Miami, FL
                    </h4>
                    <p>
                        Creative Direction, User Experience, Visual Design,
                        Project Management, Team Leading
                    </p>
                </VerticalTimelineElement>
                <VerticalTimelineElement
                    className='vertical-timeline-element--work'
                    contentStyle={{
                        background: 'white',
                        color: 'black',
                    }}
                    contentArrowStyle={{
                        borderRight: '7px solid  white',
                    }}
                    date='Oct-Nov 2020'
                    iconStyle={{
                        background: 'white',
                    }}
                    icon={<i class='fas fa-project-diagram projectIcon'></i>}
                >
                    <h3 className='vertical-timeline-element-title'>Madlibz</h3>
                    <h4 className='vertical-timeline-element-subtitle'>
                        Miami, FL
                    </h4>
                    <p>
                        Creative Direction, User Experience, Visual Design,
                        Project Management, Team Leading
                    </p>
                </VerticalTimelineElement>
            </VerticalTimeline>
        </>
    );
};

export default Projects;
