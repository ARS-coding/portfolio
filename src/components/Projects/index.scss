#projects {
  margin-top: 50px;
}

.projects-container {
  width: 100%;
  .swiper-wrapper {
    height: 1300px !important;
  }
}
