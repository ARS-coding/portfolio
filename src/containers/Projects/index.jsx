import React from 'react';
import { Container } from 'react-bootstrap';
import { Swiper, SwiperSlide } from 'swiper/react';
import ProjectCard from '../../components/ProjectCard';
import dataArr from './data';
import './index.scss';

const pagination = {
  clickable: true,
  renderBullet: (index, className) => `<div class="${className}"></div>`,
  draggable: true,
};

const SWIPER_BREAKPOINTS = {
  350: {
    slidesPerView: 1,
    spaceBetween: 5,
  },
  380: {
    slidesPerView: 1.1,
    spaceBetween: 10,
  },
  450: {
    slidesPerView: 1.2,
    spaceBetween: 10,
  },
  576: {
    slidesPerView: 1.2,
    spaceBetween: 10,
  },
  768: {
    slidesPerView: 1.5,
    spaceBetween: 15,
  },
  992: {
    slidesPerView: 2.1,
    spaceBetween: 15,
  },
  1200: {
    slidesPerView: 2.2,
    spaceBetween: 20,
  },
  1400: {
    slidesPerView: 2.5,
    spaceBetween: 30,
  },
};

const swipers = dataArr.map((data) => (
  <SwiperSlide key={data.title}>
    <ProjectCard
      title={data.title}
      type={data.type}
      githubUrl={data.githubUrl}
      demoUrl1={data.demoUrl1}
      demoUrl2={data.demoUrl2}
      goal={data.goal}
      impactArr={data.impactArr}
      skillsArr={data.skillsArr}
      responsibilityArr={data.responsibilityArr}
      iosUrl={data.iosUrl}
      androidUrl={data.androidUrl}
      status={data.status}
    />
  </SwiperSlide>
));

const Projects = () => (
  <Container>
    <h2 id="projects">Projects</h2>
    <Swiper
      pagination={pagination}
      className="swiper-container"
      break
      breakpoints={SWIPER_BREAKPOINTS}
    >
      {swipers}
    </Swiper>
  </Container>
);

export default Projects;
