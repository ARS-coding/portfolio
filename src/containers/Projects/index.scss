.swiper-container {
  margin-top: 20px;
}
