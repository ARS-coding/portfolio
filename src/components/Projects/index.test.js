import React from 'react';
import renderer from 'react-test-renderer';
import Projects from '.';

it('Projects renders correctly', () => {
    const tree = renderer.create(<Projects />).toJSON();
    expect(tree).toMatchSnapshot();
});
