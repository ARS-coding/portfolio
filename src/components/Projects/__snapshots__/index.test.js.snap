// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects renders correctly 1`] = `
<div
  className="container"
>
  <h1
    className="homeTitle"
    id="projects"
  >
    Projects
  </h1>
  <div
    className="vertical-timeline vertical-timeline--two-columns"
  >
    <div
      className="vertical-timeline-element--work vertical-timeline-element"
      id=""
      style={null}
    >
      <span
        className="vertical-timeline-element-icon is-hidden"
        onClick={null}
        style={
          Object {
            "background": "white",
          }
        }
      >
        <i
          className="fas fa-project-diagram projectIcon"
        />
      </span>
      <div
        className="vertical-timeline-element-content is-hidden"
        onClick={null}
        style={
          Object {
            "background": "white",
            "color": "white",
          }
        }
      >
        <div
          className="vertical-timeline-element-content-arrow"
          style={
            Object {
              "borderRight": "7px solid  white",
            }
          }
        />
        <div
          className="homeText projectContent"
        >
          <h4>
            Juniper
          </h4>
          <a
            href="https://github.com/ReCoded-Org/capstone-istanbul-juniper"
            rel="noreferrer"
            target="_blank"
          >
            Github
          </a>
          <br />
          <a
            href="https://juniper-ist.netlify.app/"
            rel="noreferrer"
            target="_blank"
          >
            Demo
          </a>
          <p>
            <strong>
              Goal:
            </strong>
             teaching middle-eastern youth about environmental problems through games.
          </p>
          <br />
          <h5>
            My Responsibilities
          </h5>
          <ul>
            <li>
              Developing a memory game
            </li>
            <li>
              Code reviewing
            </li>
            <li>
              Bug bashing
            </li>
            <li>
              Developing games page
            </li>
            <li>
              Scrum master for 1 week
            </li>
            <li>
              Helping colleagues whenever needed
            </li>
            <li>
              Developing twitter section of homepage
            </li>
          </ul>
          <h5>
            Skills Applied
          </h5>
          <ul>
            <li>
              HTML
            </li>
            <li>
              CSS
            </li>
            <li>
              JavaScript
            </li>
            <li>
              React
            </li>
            <li>
              Git and Github
            </li>
            <li>
              AGILE Methodologies
            </li>
            <li>
              Teamwork
            </li>
            <li>
              Firebase
            </li>
            <li>
              Ant Design
            </li>
            <li>
              Responsive Design
            </li>
            <li>
              Figma
            </li>
            <li>
              Testing (Jest)
            </li>
            <li>
              Routing
            </li>
            <li>
              i18n
            </li>
          </ul>
        </div>
        <span
          className="vertical-timeline-element-date"
        >
          
        </span>
      </div>
    </div>
    <div
      className="vertical-timeline-element--work vertical-timeline-element"
      id=""
      style={null}
    >
      <span
        className="vertical-timeline-element-icon is-hidden"
        onClick={null}
        style={
          Object {
            "background": "white",
          }
        }
      >
        <i
          className="fas fa-project-diagram projectIcon"
        />
      </span>
      <div
        className="vertical-timeline-element-content is-hidden"
        onClick={null}
        style={
          Object {
            "background": "white",
            "color": "white",
          }
        }
      >
        <div
          className="vertical-timeline-element-content-arrow"
          style={
            Object {
              "borderRight": "7px solid  white",
            }
          }
        />
        <div
          className="homeText projectContent"
        >
          <h4>
            FoodCheck
          </h4>
          <a
            href="https://github.com/tmansuroglu/MohammadJundi-Tarkan-Project"
            rel="noreferrer"
            target="_blank"
          >
            Github
          </a>
          <br />
          <a
            href="https://foodcheck.netlify.app/"
            rel="noreferrer"
            target="_blank"
          >
            Demo
          </a>
          <p>
            <strong>
              Goal:
            </strong>
             creating a quick and easy to use weekly diet and calory tracker
          </p>
          <br />
          <h5>
            My Responsibilities
          </h5>
          <ul>
            <li>
              App planning
            </li>
            <li>
              Setting up redux
            </li>
            <li>
              Setting up firebase
            </li>
            <li>
              Setting up food API
            </li>
            <li>
              Diet management page
            </li>
            <li>
              Creating charts
            </li>
            <li>
              Responsive design
            </li>
            <li>
              Home page
            </li>
            <li>
              Contact page
            </li>
            <li>
              Login page
            </li>
            <li>
              Register page
            </li>
          </ul>
          <h5>
            Skills Applied
          </h5>
          <ul>
            <li>
              HTML
            </li>
            <li>
              CSS
            </li>
            <li>
              JavaScript
            </li>
            <li>
              React
            </li>
            <li>
              Redux
            </li>
            <li>
              API Usage
            </li>
            <li>
              Firebase
            </li>
            <li>
              Ant Design
            </li>
            <li>
              Responsive Design
            </li>
            <li>
              Git and Github
            </li>
            <li>
              Chart.js
            </li>
            <li>
              Routing
            </li>
          </ul>
        </div>
        <span
          className="vertical-timeline-element-date"
        >
          
        </span>
      </div>
    </div>
    <div
      className="vertical-timeline-element--work vertical-timeline-element"
      id=""
      style={null}
    >
      <span
        className="vertical-timeline-element-icon is-hidden"
        onClick={null}
        style={
          Object {
            "background": "white",
          }
        }
      >
        <i
          className="fas fa-project-diagram projectIcon"
        />
      </span>
      <div
        className="vertical-timeline-element-content is-hidden"
        onClick={null}
        style={
          Object {
            "background": "white",
            "color": "white",
          }
        }
      >
        <div
          className="vertical-timeline-element-content-arrow"
          style={
            Object {
              "borderRight": "7px solid  white",
            }
          }
        />
        <div
          className="homeText projectContent"
        >
          <h4>
            Portfolio
          </h4>
          <a
            href="https://github.com/tmansuroglu/portfolio"
            rel="noreferrer"
            target="_blank"
          >
            Github
          </a>
          <br />
          <p>
            <strong>
              Goal:
            </strong>
             building a portfolio website to create an online presence.
          </p>
          <br />
          <h5>
            My Responsibilities
          </h5>
          <ul>
            <li>
              Solo Project
            </li>
          </ul>
          <h5>
            Skills Applied
          </h5>
          <ul>
            <li>
              HTML
            </li>
            <li>
              CSS
            </li>
            <li>
              JavaScript
            </li>
            <li>
              React
            </li>
            <li>
              Emailjs
            </li>
            <li>
              Bootstrap
            </li>
            <li>
              Responsive Design
            </li>
            <li>
              Git and Github
            </li>
            <li>
              Testing (Jest)
            </li>
          </ul>
        </div>
        <span
          className="vertical-timeline-element-date"
        >
          
        </span>
      </div>
    </div>
    <div
      className="vertical-timeline-element--work vertical-timeline-element"
      id=""
      style={null}
    >
      <span
        className="vertical-timeline-element-icon is-hidden"
        onClick={null}
        style={
          Object {
            "background": "white",
          }
        }
      >
        <i
          className="fas fa-project-diagram projectIcon"
        />
      </span>
      <div
        className="vertical-timeline-element-content is-hidden"
        onClick={null}
        style={
          Object {
            "background": "white",
            "color": "white",
          }
        }
      >
        <div
          className="vertical-timeline-element-content-arrow"
          style={
            Object {
              "borderRight": "7px solid  white",
            }
          }
        />
        <div
          className="homeText projectContent"
        >
          <h4>
            TMDB
          </h4>
          <a
            href="https://github.com/tmansuroglu/TMDB-Project"
            rel="noreferrer"
            target="_blank"
          >
            Github
          </a>
          <br />
          <a
            href="https://tmdbproject.netlify.app/"
            rel="noreferrer"
            target="_blank"
          >
            Demo
          </a>
          <p>
            <strong>
              Goal:
            </strong>
             creating an IMDB clone.
          </p>
          <br />
          <h5>
            My Responsibilities
          </h5>
          <ul>
            <li>
              Setting up TMDB API
            </li>
            <li>
              Movie/Tv Series Page
            </li>
            <li>
              Actor/Actress Page
            </li>
            <li>
              Search Page
            </li>
            <li>
              Main Page
            </li>
            <li>
              Navbar Dropdowns
            </li>
            <li>
              Visual Design
            </li>
          </ul>
          <h5>
            Skills Applied
          </h5>
          <ul>
            <li>
              HTML
            </li>
            <li>
              CSS
            </li>
            <li>
              JavaScript
            </li>
            <li>
              Bootstrap
            </li>
            <li>
              Responsive Design
            </li>
            <li>
              API Usage
            </li>
          </ul>
        </div>
        <span
          className="vertical-timeline-element-date"
        >
          
        </span>
      </div>
    </div>
    <div
      className="vertical-timeline-element--work vertical-timeline-element"
      id=""
      style={null}
    >
      <span
        className="vertical-timeline-element-icon is-hidden"
        onClick={null}
        style={
          Object {
            "background": "white",
          }
        }
      >
        <i
          className="fas fa-project-diagram projectIcon"
        />
      </span>
      <div
        className="vertical-timeline-element-content is-hidden"
        onClick={null}
        style={
          Object {
            "background": "white",
            "color": "white",
          }
        }
      >
        <div
          className="vertical-timeline-element-content-arrow"
          style={
            Object {
              "borderRight": "7px solid  white",
            }
          }
        />
        <div
          className="homeText projectContent"
        >
          <h4>
            Madlibz
          </h4>
          <a
            href="https://github.com/tmansuroglu/Madlibzz"
            rel="noreferrer"
            target="_blank"
          >
            Github link
          </a>
          <br />
          <a
            href="https://madlibz.netlify.app/"
            rel="noreferrer"
            target="_blank"
          >
            Demo
          </a>
          <p>
            <strong>
              Goal:
            </strong>
             creating a Madlibz game.
          </p>
          <br />
          <h5>
            My Responsibilities
          </h5>
          <ul>
            <li>
              Game Logic
            </li>
            <li>
              Setting up i18n
            </li>
            <li>
              Visual Design
            </li>
            <li>
              Responsive Design
            </li>
          </ul>
          <h5>
            Skills Applied
          </h5>
          <ul>
            <li>
              HTML
            </li>
            <li>
              CSS
            </li>
            <li>
              JavaScript
            </li>
            <li>
              Regular Expression
            </li>
            <li>
              i18n
            </li>
            <li>
              Bootstrap
            </li>
            <li>
              Visual Design
            </li>
            <li>
              Responsive Design
            </li>
            <li>
              Teamwork
            </li>
          </ul>
        </div>
        <span
          className="vertical-timeline-element-date"
        >
          
        </span>
      </div>
    </div>
    <div
      className="vertical-timeline-element--work vertical-timeline-element vertical-timeline-element--no-children"
      id=""
      style={null}
    >
      <span
        className="vertical-timeline-element-icon is-hidden"
        onClick={null}
        style={
          Object {
            "background": "white",
          }
        }
      >
        <i
          className="fas fa-play startIcon"
        />
      </span>
      <div
        className="vertical-timeline-element-content is-hidden"
        onClick={null}
        style={
          Object {
            "color": "white",
          }
        }
      >
        <div
          className="vertical-timeline-element-content-arrow"
          style={
            Object {
              "borderRight": "7px solid  white",
            }
          }
        />
        
        <span
          className="vertical-timeline-element-date"
        >
          
        </span>
      </div>
    </div>
  </div>
</div>
`;
